packages/backend/src/misc/is-duplicate-key-value-error.ts

```diff
@@ -1,3 +1,7 @@
 export function isDuplicateKeyValueError(e: unknown | Error): boolean {
-	return (e as any).message && (e as Error).message.startsWith('duplicate key value');
+	return (
+		(e as any).message &&
+		((e as Error).message.startsWith("duplicate key value") ||
+			(e as Error).message.startsWith("重複したキー値"))
+	);
```

Comment:
pg driverのエラーコードをそのまま取ってこれるんじゃないかなと (動作は未確認)
https://github.com/brianc/node-postgres/blob/eaafac36dc8f4a13f1fecc9e3420d35559fd8e2b/packages/pg-protocol/src/messages.ts#L97-L117

```
  driverError:
    code: '23505',
    constraint: 'IDX_ad0c221b25672daf2df320a817',
```

```suggestion
	return e instanceof QueryFailedError && e.driverError.code === '23505';
```

Reply:
ちょっと検証してみますね。

Reply:
いかすきーで動作確認が取れたので、上記変更してコミットかけました。何卒。